cypress/integration/dresses-list.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { MenuContentPage, DressesListPage } from "../page/index"
+
+
+describe("the user navigates to the dresses page should", () => {
+
+  let menuContentPage: MenuContentPage;
+  let dressesListPage: DressesListPage;
+
```

Comment:
you can create here an array with the expected list of dresses. As part of the arrange section